go/libraries/doltcore/sqle/database_provider.go

```diff
@@ -279,6 +281,15 @@ func (p DoltDatabaseProvider) AllDatabases(ctx *sql.Context) (all []sql.Database
 			foundDatabase = true
 		}
 		all = append(all, db)
+
+		if showBranches {
+			revisionDbs, err := p.allRevisionDbs(ctx, db)
+			if err != nil {
+				// TODO: this interface is wrong, needs to return errors
```

Comment:
Probably still worth logging a warning or error even if we can't return it here.